app/components/link-to-username/component.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+const {
+  computed
+} = Ember;
+
+export default Ember.Component.extend({
+  username: null,
+
+  role: computed('username', function() {
+    if (this.get('username').startsWith('@')) {
+      return 'op';
+    } else {
```

Comment:
`+` is voice, not moderator (meaning you can talk when the channel is set to silent, when only voices and ops can talk). `%` is half op, which is close to moderator (they can voice and kick/ban people with a lower level than themselves.

I'm adding support for voice and half op